lib/cosmos/gui/line_graph/line_graph_scaling.rb

```diff
@@ -275,14 +275,6 @@ def build_x_grid_lines
       @x_grid_lines = []
 
       @x_grid_line_scale = determine_grid_line_scale(@manual_x_grid_line_scale, @x_min, @x_max)
-      if @manual_x_grid_line_scale
-        @x_grid_line_scale = @manual_x_grid_line_scale
-      else
-        diff = @x_max - @x_min
-        @x_grid_line_scale = calculate_base(diff)
-      end
-      @x_grid_line_scale = 1.0 if @x_grid_line_scale < 1
```

Comment:
I explicitly removed this line as I think the x axis should auto scale. For example if you're graphing 100Hz data for a small time period.